basket.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
 
   # Uncomment to register a new dependency of your gem
   spec.add_dependency "redis"
+  spec.add_dependency "redis-namespace"
```

Comment:
@nicholalexander — let me know what you think of this. It's another dependency, though I felt it was worth it to remove any unexpected key access.

Reply:
i like this!  i think i had looked at this with you, no?  i also ran across this based on a link that was shared in slack the other day: https://github.com/sidekiq/sidekiq/blob/3b93c250d08df79e9296ade0c0e15d7e512035af/lib/sidekiq/api.rb#L229

i'm good to add it!